internal/driver/interactive.go

```diff
@@ -267,6 +273,9 @@ func parseCommandLine(input []string) ([]string, variables, error) {
 		}
 	}
 	if c == nil {
+		if v := pprofVariables[name]; v != nil {
+			return nil, nil, fmt.Errorf("do you mean: \"%v=%v\"?", name, args[0])
```

Comment:
Use backtick string to avoid having to escape double quotes.

Reply:
Done.